corehq/apps/cachehq/cachemodels.py

```diff
@@ -76,7 +76,6 @@ class LocationGenerationCache(GenerationCache):
     generation_key = '#gen#location#'
     doc_types = ['Location']
     views = [
-        'commtrack/locations_by_code',
         '_all_docs',
```

Comment:
sorry let me state that clearer. if `Location` objects are still using generational caching, _and_ somewhere that generational caching is getting passed through to a query that hits the `_all_docs` view, then I think that would need to get fixed elsewhere (manually invalidated).

Thinking about it a bit - I don't think it could be using `iter_docs` because that doesn't support passing in the model (which is where caching would happen), so I think as long as there are nothing that calls `Location.view('_all_docs')` then this is fine to remove.